### ISSUE TITLE
TN_MaintenanceAuthority-OwnerAuthority_CI_Citation not matching

### ISSUE THREAD

**Issue by JohannaOtt** [INSPIRE-MIF/application-schemas#60]
Edit: this was to clarify and is answered with the comments:

> Not yet a change proposal but rather a question:
> I seem to remember that there was a discussion to replace CI_Citation as type for MaintenanceAuthority.authority because it does not really match the purpose. But I cannot find any documentation or disccusion on this any more.
> The only thing I can find is [this discussion](https://wayback.archive-it.org/12090/20210120054906/https://inspire.ec.europa.eu/forum/discussion/view/39163/filling-the-attribute-maintenanceauthority) on how to fill it but that does not address changing the schema.
> Is anyone aware of such a discussion?
> 
> Thank you and all the best
> 
> Johanna

# Change proposal description
Change the type used for MaintenanceAuthority.authority and OwnerAuthority.authority from gmd:CI_Citation to CI_ResponsibleParty

## Issue faced
The type used for the attributes does not match their description.

### Expected behaviour
The attributes should use CI_ResponsibleParty (defined as "identification of, and means of communication with, person(s) and organizations associated with the dataset")

### Current behaviour
The two attributes mentioned (defined as "Identification of the maintenance authority." and "Identification of the owning authority.") are currently using gmd:CI_Citation (defined as "standardized resource reference") as type. 

## Proposed solution
Change the type for MaintenanceAuthority.authority and OwnerAuthority.authority from gmd:CI_Citation to CI_ResponsibleParty.

# Additional information

## Proposal reason
This change proposal is needed because the schema model does not fit the purpose.

## Addressed schema
[TN](https://inspire.ec.europa.eu/schemas/tn/4.0/CommonTransportElements.xsd)

## Impact on INSPIRE TG / IR
IR on interoperability:
* 7.3.1.3 : Change Type in table from CI_Citation to CI_ResponsibleParty
* 7.3.1.5 : Change Type in table from CI_Citation to CI_ResponsibleParty

TG TN
* UML diagrams
* 5.3.2.1.3 Value type: CI_ResponsibleParty
* 5.3.2.1.5 Value type: CI_ResponsibleParty

## Change proposer
Johanna Ott, wetransform GmbH

### More detailed documentation
[Discussion in Forum](https://wayback.archive-it.org/12090/20210120054906/https://inspire.ec.europa.eu/forum/discussion/view/39163/filling-the-attribute-maintenanceauthority)

**Comment by fabiovinci** [comment]
Dear @JohannaOtt,

I don’t remember any other discussions on the subject.
In any case, I think you can complete your change proposal in order to start the governance process.

Have you thought about the alternative dataType? It could be "_CI_ResponsibleParty_".

Moreover, I think that the same problem also concerns the attribute "_authority_" of the feature type "_OwnerAuthority_".

**Comment by sMorrone** [comment]
@JohannaOtt 
since the changing of the authority data type would require a change in the IR (and clearly also in the TG, the UML and the XML schema itself), I just checked the document with latest IR change proposals (now in the Comitology procedure) and there is no change request for the MaintenanceAuthority / OwnerAuthority, hence for sure the Forum discussion did not turn into any IR change request.  That said, I agree that the CI_Citation data type, meant for 'standardized resource reference' is not suited to authority element and agree with @fabiovinci that Responsible Party could be more suited.

**Comment by JohannaOtt** [comment]
@fabiovinci @sMorrone Thank you for the feedback. I adapted the proposal description accordingly.

**Comment by fabiovinci** [comment]
Below are some screenshots of the involved elements.

![image](https://user-images.githubusercontent.com/22212406/158166226-df2cf0b6-207d-455e-a6f8-a10735d6d86b.png)

![image](https://user-images.githubusercontent.com/22212406/158166504-695ef69f-01b5-4508-9f3f-ff447e7c4a21.png)

![image](https://user-images.githubusercontent.com/22212406/158166515-d3c8938f-5362-4876-9482-6b7001e8f2c4.png)

**Comment by sMorrone** [comment]
Subgroup meeting on 24.03.2022:
the Subgroup approved this change proposal.

**Comment by idevisser** [comment]
I agree CI_ResponsibleParty suits better,but I wonder what the impact is on existing implementations using the gmd:CI_Citation.

**Comment by sMorrone** [comment]
If the change request is approved, which may not be so soon as the proposed change will follow the established procedure for proposing changes to the IR, the existing implementations would certainly be impacted, as the proposed one is a breaking change preventing the backward compatibility of the TN schema.

**Comment by idevisser** [comment]
@sMorrone thanx, can you add this label wherever it applies for the issues on the agenda of the MIG-T ?

**Comment by sMorrone** [comment]
During the joint MIG/MIG-T meeting held on 31/03-01/04/2022, the proposal was accepted. However, a request for additional information was put forward by AT and it was agreed that a formal request for clarification will be added as a comment to this thread. Once this will be addressed, the proposal can be endorsed without the need for additional voting.
More details in the meeting minutes on the [69th MIG-T meeting page](https://wikis.ec.europa.eu/display/InspireMIG/69th+MIG-T+meeting+2022-04-01)

**Comment by sMorrone** [comment]
During the 71st MIG-T held on 2022-10-07, AT was asked to provide the expected feedback by 31 Oct 2022, after which date, in the absence of feedback, the workflow on the proposal (already approved by the MIG) would have been considered completed (see [meeting minutes](https://wikis.ec.europa.eu/display/InspireMIG/71st+MIG-T+meeting+2022-10-07), under “_Action 2.3.1. Governance of INSPIRE artefacts_”). Given the absence of feedback as of today's date, this proposal is considered approved by the MIG.
